openfasoc/generators/glayout/glayout/flow/blocks/elementary/FVF/README.md

```diff
@@ -0,0 +1,37 @@
+## FLIPPED VOLTAGE FOLLOWER CELL
```

Comment:
It would be good to include some more documentation such as a schematic, some code examples, or layout images.